lib/savon/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module Savon
-  VERSION = '3.0.0.rc1'
+  VERSION = '3.0.0.rc2'
```

Comment:
Minor: Avoid changing the version in this PR. The release process takes care of changing that.